docs/content/help/contact_us.md

```diff
@@ -0,0 +1,16 @@
+# Contact Us
+
+## Mailing Lists
+
+The best way to contact the MOOSE team is to utilize the mailing list:
+
+* MOOSE Users for technical Q&A (moderate traffic): [moose-users@googlegroups.com](https://groups.google.com/forum/#!forum/moose-users)
+* MOOSE Announcements (very light traffic): [moose-announce@googlegroups.com](https://groups.google.com/forum/#!forum/moose-announce)
+
+!!! note "Gmail Users"
+    Just click the "Join group" button
+
+!!! note "Non-Gmail Users"
+    Send an email to:
+    * [moose-users+subscribe@googlegroups.com](mailto:moose-users+subscribe@googlegroups.com) and/or
+    * [moose-announce+subscribe@googlegroups.com](mailto:moose-announce+subscribe@googlegroups.com)
```

Comment:
The default theme is hard to read. Its dark blue links on top of a light blue admonition. I know there is no way around this currently however. Just pointing it out. Its even worse when you use dorkmode ;)